### PR TITLE
feat: adds new ui

### PR DESCRIPTION
**Pull request by rinzler99**

### CODE REVIEW

File: examples/vue-app/src/components/HelloWorld.vue [hunk 1]
```diff
@@ -356,216 +390,347 @@ const topup = async () => {
 };
 
 const debugConsole = async (text: string) => {
-  document.querySelector("#console > p")!.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
+  // document.querySelector("#console > p")!.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
+  consoleDiv.value.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
```
Comment (guru-web3) [inline examples/vue-app/src/components/HelloWorld.vue:394]: if(consoleDiv.value)
    consoleDiv.value.innerHTML = typeof text === "object" ? JSON.stringify(text) : text;
    
    use this, ts error is being throwed in local IDE

File: examples/vue-app/src/components/HelloWorld.vue [hunk 2]
```diff
@@ -356,216 +390,347 @@ const topup = async () => {
...
+function getAddress(address: string) {
+  if (address.length < 11) {
+    return address;
+  }
+  if (typeof address !== "string") return "";
+  return `${address.slice(0, 5)}...${address.slice(-5)}`;
+}
+const clearUiconsole = (): void => {
+  consoleDiv.value.innerHTML = "";
+};
```
Comment (guru-web3) [inline examples/vue-app/src/components/HelloWorld.vue:569]: if(consoleDiv.value)
    consoleDiv.value.innerHTML = ""

File: examples/vue-app/package.json [hunk 3]
```diff
@@ -13,9 +13,10 @@
     "@project-serum/borsh": "^0.2.3",
     "@solana/spl-token": "^0.1.8",
     "@solana/web3.js": "^1.34.0",
-    "@toruslabs/solana-embed": "file:../../toruslabs-solana-embed-0.2.0.tgz",
+    "@toruslabs/solana-embed": "file:../../",
```
Comment (guru-web3) [inline examples/vue-app/package.json:16]: undo this

File: examples/vue-app/postcss.config.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
```
Comment (guru-web3) [inline examples/vue-app/postcss.config.js:3]: we cana define color theme here #595857 and use it in hellworld.vue

File: examples/vue-app/src/components/HelloWorld.vue [hunk 5]
```diff
@@ -73,7 +75,7 @@ const login = async () => {
       //   },
       // });
     }
-    console.log(torus)
+    // console.log(torus);
```
Comment (guru-web3) [inline examples/vue-app/src/components/HelloWorld.vue:78]: we need consoles here

File: examples/vue-app/src/components/HelloWorld.vue [hunk 6]
```diff
@@ -356,216 +390,347 @@ const topup = async () => {
...
-        Build Environment :
-        <i>{{ buildEnv }}</i>
-      </p>
-      <p v-if="network" class="break-word">Solana Network : {{ network }}</p>
-      <p v-if="pubkey" class="break-word">Publickey : {{ pubkey }}</p>
-      <div v-if="pubkey === ''">
-        <select name="buildEnv" v-model="buildEnv">
+    <div class="grid text-center justify-center pt-20" v-if="!pubkey">
+      <h7 class="font-bold text-3xl">Login and resets</h7>
+      <h6 class="font-semibold text-[#595857]">Build Environment : {{ buildEnv }}</h6>
```
Comment (guru-web3) [inline examples/vue-app/src/components/HelloWorld.vue:576]: can we move this #595857 to tailwind config

**Review by guru-web3 — CHANGES_REQUESTED** [review]

File: examples/vue-app/src/components/HelloWorld.vue (hunk 1 above)
Comment (rinzler99) [reply examples/vue-app/src/components/HelloWorld.vue:394]: resolved

**Review by guru-web3 — APPROVED** [review]
lgtm

**Comment by rinzler99** [comment]
Link for the working UI : https://www.loom.com/share/d61ce7447d4249ca95e561f8a5bdbf1e

Figma File : https://www.figma.com/file/3p3IoglxHHlry87ojz8htF/Torus-Wallet?node-id=1201%3A11189

**Pull request merged**